var/spack/repos/builtin/packages/llvm-amdgpu/package.py

```diff
@@ -23,14 +23,16 @@ class LlvmAmdgpu(CMakePackage):
 
     variant('build_type', default='Release', values=("Release", "Debug"), description='CMake build type')
 
-    depends_on('cmake@3:', type='build')
+    depends_on('cmake@3.4.3:',  type='build', when='@:3.8.99')
+    depends_on('cmake@3.13.4:', type='build', when='@3.9.0:')
```

Comment:
while building llvm-amdgpu with the  latest spack:develop, i see build failure. the error seems to be due to package using the cmake@3.19.0 . When i use cmake@3.18.4 instead ,  i am able to build llvm-amdgpu@3.9.0 as well as llvm-amdgpu@3.8.0 . Can you please make this change as it will pick cmake@3.19.0 with the above change ?

Reply:
Yeah, I would also just blacklist cmake 3.19. Upstream they reported it's caused by LLVM abusing cmake, so another option is to patch LLVM, but ... we have enough patches already :sweat: